### PR TITLE
add Time Door

### PR DESCRIPTION
**Pull request by functionalize**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,6 +160,7 @@ Table of Contents
   * [Beeceptor](https://beeceptor.com) - Mock a rest API in seconds, fake API response and much more. Free 50 requests per day, public dashboard, open endpoints (anyone having link to the dashboard can view requests and responses).
   * [ScreenshotAPI.net](https://screenshotapi.net/) - Screenshot API use one simple API call to generate screenshots of any website. Build to scale and hosted on Google Cloud. Offers 100 free screenshots per month.
     * [shrtcode API](https://shrtco.de/docs) - Free URL Shortening API without authorisation and no request limits
+  * [Time Door](https://timedoor.io) - A time series analysis API
```
Comment (ripienaar) [inline README.md]: Whats the free offering here? I do not see anything mentioned on their page. Fair enough its ridiculously cheap, but still it doesnt seem free :)

Comment (functionalize) [reply README.md]: free offering is `100000ms free compute`

Comment (ripienaar) [reply README.md]: Ah I see it now, thanks!

**Comment by ripienaar** [comment]
would you mind rebasing the PR? Someone sorted this section and its now very hard to resolve easily :(

**Comment by functionalize** [comment]
> would you mind rebasing the PR? Someone sorted this section and its now very hard to resolve easily :(

Sure, I hope I did it correctly.

**Comment by functionalize** [comment]
> would you mind rebasing the PR? Someone sorted this section and its now very hard to resolve easily :(

k, done.

**Comment by ripienaar** [comment]
thank you

**Pull request merged**